src/transports/ipc/cipc.c

```diff
@@ -315,7 +315,7 @@ static void nn_cipc_handler (struct nn_fsm *self, int src, int type,
 /******************************************************************************/
 /*  WAITING state.                                                            */
 /*  Waiting before re-connection is attempted. This way we won't overload     */
-/*  the system by continuous re-connection attemps.                           */
+/*  the system by continuous re-connection attempts.                           */
```

Comment:
Can you fix the alignment of the asterisk?  I hate this kind of block style comment -- but this code was written by Martin.

Reply:
Thanks @gdamore done